daemon/create_test.go

```diff
@@ -17,5 +16,5 @@ func TestVerifyNetworkingConfig(t *testing.T) {
 		EndpointsConfig: endpoints,
 	}
 	err := verifyNetworkingConfig(nwConfig)
-	assert.Check(t, errdefs.IsInvalidParameter(err))
+	assert.Check(t, err != nil)
```

Comment:
While we're updating, perhaps we should check for the correct error WDYT?

```suggestion
		assert.Check(t, is.Error(err, "no EndpointSettings for mynet"))
```

Reply:
That's what I did at first, but then realized @cpuguy83 was against it: https://github.com/moby/moby/pull/36077#discussion_r162946234